ruby/bitemporality.gemspec

```diff
@@ -0,0 +1,43 @@
+lib = File.expand_path("../lib", __FILE__)
```

Comment:
Is this file handwritten? I'm guessing not based on the funkiness of its contents -> what command did you run to generate it?

It seems like `bundle gem <gem-name>` can auto-generate one, but the PR doesn't include other things like a license file.

Reply:
I wanted the change to be as small as possible, so I just cherry-picked in this change from a `bundle gem` command I ran elsewhere.